openapi.yaml

```diff
@@ -3062,6 +3062,855 @@ paths:
+    x-linode-cli-command: databases
+    parameters:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+      - name: backupId
+        in: path
+        description: The ID of the Managed MySQL Database backup.
+        required: true
+        schema:
+          type: integer
+    get:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup View
+      operationId: getDatabasesMySQLInstanceBackup
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-view
+      x-linode-grant: unrestricted only
+      description: |
+        Display information for a single backup for an accessible Managed MySQL Database on your Account.
+
+        Only unrestricted Users can access this command.
+
+        **Beta**: This endpoint is in **closed** beta. To get early access and test new Linode products before they
+        hit the market, join the [Linode Green Light](https://www.linode.com/green-light/) program.
+        When accessing this endpoint, please make sure to prepend all requests with `/v4beta` instead of `/v4`, and be
+        aware that this endpoint may receive breaking updates in the future.  This notice will be removed when this
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - databases:read_write
+      responses:
+        '200':
+          description: Returns a single backup for the Managed MySQL Database.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatabaseBackup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4beta/databases/mysql/instances/123/backups/456
+      - lang: CLI
+        source: >
+          linode-cli databases mysql-backup-view 123 456
+  /databases/mysql/instances/{instanceId}/backups/{backupId}/restore:
+    x-linode-cli-command: databases
+    parameters:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+      - name: backupId
+        in: path
+        description: The ID of the Managed MySQL Database backup.
+        required: true
+        schema:
+          type: integer
+    post:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup Restore
+      operationId: postDatabasesMySQLInstanceBackupRestore
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-restore
+      x-linode-grant: unrestricted only
+      description: |
+        Restore a backup to an active Managed MySQL Database on your Account.
+
+        Only unrestricted Users can access this command.
```

Comment:
Definitely a benefit if it makes it clearer beyond stating "an active Managed MySQL Database". I'll move the requirement to its own line so that it stands out more on all commands that require an active state.